projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -33,11 +35,13 @@ export class TabParagraphContainerComponent
   tabTitleParams: Observable<any>[] = [];
 
   subscription: Subscription;
+  tabSubscription: Subscription;
 
   constructor(
     public componentData: CmsComponentData<CMSTabParagraphContainer>,
     protected cmsService: CmsService,
-    protected winRef: WindowRef
+    protected winRef: WindowRef,
+    protected breakpointService: BreakpointService
```

Comment:
Can we make it backward compatible? It's a new constructor's required parameter.